select.go

```diff
@@ -153,6 +159,42 @@ func (b *SelectStmt) Build(d Dialect, buf Buffer) error {
 	return nil
 }
 
+// https://docs.microsoft.com/en-us/previous-versions/sql/sql-server-2012/ms188385(v=sql.110)
+func (b *SelectStmt) addMSSQLLimits(buf Buffer) {
+	limitCount := b.LimitCount
+	offsetCount := b.OffsetCount
+	if limitCount < 0 && offsetCount < 0 {
+		return
+	}
+	if offsetCount < 0 {
+		offsetCount = 0
+	}
+
+	if len(b.Order) == 0 {
```

Comment:
Return if len(b.Column) == 0

Reply:
Yes. This condition already exists in "Build" function
```
func (b *SelectStmt) Build(d Dialect, buf Buffer) error {
	if b.raw.Query != "" {
		return b.raw.Build(d, buf)
	}

	if len(b.Column) == 0 {
		return ErrColumnNotSpecified
	}

...

	if d == dialect.MSSQL {
		b.addMSSQLLimits(buf)
	} else {
...
}
```